include/tvm/relay/transform.h

```diff
@@ -231,11 +232,34 @@ Pass CreateModulePass(
  * \return The created function pass.
  */
 Pass CreateFunctionPass(
-    const runtime::TypedPackedFunc<Function(Function, PassContext)>& pass_func,
+    const runtime::TypedPackedFunc<Function(Function, Module, PassContext)>& pass_func,
     int opt_level,
     const std::string& name,
     const tvm::Array<tvm::Expr>& required);
 
+Pass DeadCodeElimination();
```

Comment:
Mark the declaration with prefix TVM_DLL so that it is exposed in the dynamic library.